Wikipedia/assets/about.html

```diff
@@ -26,6 +26,15 @@
                 color:#000000;
```

Comment:
These changes need to be made to `/www/about.html` or they'll be overwritten when we run `make grunt`.

Everything in `/assets/` gets overwritten each time we run `make grunt`.